internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go

```diff
@@ -51,9 +57,64 @@ func (m *DynamicScaleUpManager) Reconcile() []error {
 	glog.Infoln("running dynamic scale up reconcile event")
 	defer m.logReconcileEventEnd()
 
-	return nil
+	var errs []error
+	errs = m.reconcileClustersForRegions()
+
+	return errs
 }
 
 func (m *DynamicScaleUpManager) logReconcileEventEnd() {
 	glog.Infoln("dynamic scale up reconcile event finished")
 }
+
+// reconcileClustersForRegions creates an OSD cluster for each supported cloud provider and region where no cluster exists.
+func (m *DynamicScaleUpManager) reconcileClustersForRegions() []error {
```

Comment:
we could consider changing the name at this stage

```suggestion
func (m *DynamicScaleUpManager) triggerClusterScaleUpPerRegion() []error {
```

Reply:
This PR is about the refactor of moving the logic from one place to another.
No changes to the behavior (aside from the one commented about the scaling type check) will be done in this PR.

Reply:
ack